package.json

```diff
@@ -11,104 +11,64 @@
   ],
   "private": true,
   "scripts": {
-    "dev": "node build/dev-server.js",
-    "start": "node build/dev-server.js",
-    "build": "node build/build.js",
-    "unit": "cross-env BABEL_ENV=test karma start test/unit/karma.conf.js --single-run",
-    "test": "npm run unit",
-    "lint": "eslint --ext .js,.vue src test/unit/specs",
-    "lint-fix": "eslint --fix --ext .js,.vue src app test/unit/specs",
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "test": "cross-env NODE_ENV=test karma start tests/unit/karma.conf.js --single-run",
+    "lint": "vue-cli-service lint --no-fix *.js app src tests",
+    "lint:fix": "vue-cli-service lint *.js app src tests",
     "deploy": "gh-pages -d dist",
     "init:app": "node util/init-app.js"
   },
   "dependencies": {
     "axios": "^0.21.4",
     "canvas-record": "^3.0.0",
+    "core-js": "^3.8.3",
     "ol": "6.4.3",
-    "proj4": "2.5.0",
-    "vue": "2.6.11",
+    "proj4": "2.8.1",
+    "vue": "2.6.14",
     "vue-i18n": "^8.24.4",
     "vuetify": "2.2.18"
   },
   "devDependencies": {
-    "@babel/core": "^7.0.0",
```

Comment:
When I created a vue-cli template app based on v5.0.8 it used the following dev-dependencies. Did I accidently not use the most recent version of vue-cli or did you tweak dependencies manually to newer versions?

```
"devDependencies": {
    "@babel/core": "^7.12.16",
    "@babel/eslint-parser": "^7.12.16",
    "@vue/cli-plugin-babel": "~5.0.0",
    "@vue/cli-plugin-eslint": "~5.0.0",
    "@vue/cli-service": "~5.0.0",
    "@vue/eslint-config-standard": "^6.1.0",
    "eslint": "^7.32.0",
    "eslint-plugin-import": "^2.25.3",
    "eslint-plugin-node": "^11.1.0",
    "eslint-plugin-promise": "^5.1.0",
    "eslint-plugin-vue": "^8.0.3",
    "vue-template-compiler": "^2.6.14"
 }
```

Reply:
No I think we are synced here, just look 6 lines down the `package.json` diff, the line you commented is a removal of `"@babel/core": "^7.0.0"` and 6 liner later, `"@babel/core": "^7.12.16"` is added as in your template.

This comes from the fact that I didn't generate an empty project from scratch and copy `Wegue` parts inside but started from `Vue-CLI v4 PR` and upgraded `Vue` components as stated in the [official migration guide](https://cli.vuejs.org/migrations/migrate-from-v4.html).

There are some discrepancies for sure, the 5.0.8 template for example still installs `"@vue/cli-plugin-babel": "~5.0.0"` as stated in your comment where the upgrade turned this automatically into `"@vue/cli-plugin-babel": "~5.0.8"`. However, those discrepancies should at least stick with what the semver syntax says. I mean ^7.0.0 and ^7.12.16 can potentially be the same as ^7.x.x says that any package < 8.x.x can be installed if required to by a dependency and ~5.0.x means < 5.1.x.

Reply:
OK, I just wanted to make sure that this hasn`t been an oversight. The dependencies all work out and if I got you right, this is what `vue upgrade` has done to it. All good here!

Reply:
Exactly, 100% correct!